src/components/DuosDatePicker.tsx

```diff
@@ -193,7 +201,7 @@ export const DuosDatePicker = (props: DUOSDatePickerProps) => {
           },
         }}
         slots={{day: WeekendFormattedDay, actionBar: CancelSelectActionBar}}
-      />
+      />}
```

Comment:
Do we have a linter standard in this repo? I would expect to see some different spacing in these changes.

Reply:
We have https://github.com/DataBiosphere/duos-ui/blob/develop/.eslintrc